tools/icu/icu_versions.json

```diff
@@ -1,3 +1,3 @@
 {
-    "minimum_icu": 63
+    "minimum_icu": 64
```

Comment:
OK, I have missed this before… but when you bump the minimum_icu, I kind of think you should also *delete* the patches that no longer apply, so:

- tools/icu/patches/54
- tools/icu/patches/55
- tools/icu/patches/58
- tools/icu/patches/62
- tools/icu/patches/63 (though I am about to suggest a new file here)

Reply:
^ This could be a followon issue if you prefer.

Reply:
OK thanks. I removed all the patch files